simulator/host_system_test.go

```diff
@@ -130,3 +130,38 @@ func TestNewHostSystem(t *testing.T) {
 		t.Fatal("expected hs.Summary.Runtime == &hs.Runtime; got !=")
 	}
 }
+
+func TestDestroyHostSystem(t *testing.T) {
+	ctx := context.Background()
+	m := ESX()
```

Comment:
Don't think real ESX allows Destroy on the 1 HostSystem.  But this is also fine to leave as-is for now.

Reply:
got it. gonna check why it passed..